jwql/website/apps/jwql/tests/test_context_processors.py

```diff
@@ -31,6 +31,7 @@
     from jwql.utils.utils import get_base_url
 
 
+@skipIf(True, "Needs updating: throws errors for missing settings and apps")
```

Comment:
I got this test to work locally by removing "users" from the list of keys, and by adding:
```
# These lines are needed in order to use the Django models in a standalone
# script (as opposed to code run as a result of a webpage request). If these
# lines are not run, the script will crash when attempting to import the
# Django models in the line below.
from django import setup
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jwql.website.jwql_proj.settings")
setup()
```

Reply:
Thanks! I'll make that change.